File: bug27_incorrect_formula_used/solved_main.js
function run() {
    let acc=0n;
    for(let n=1n;n<=10n;n++){
        let triangle=n*(n+1n)/2n;   // FIX: was /3n; correct triangle formula is n*(n+1)/2
        acc^=BigInt.asUintN(64,triangle*0x9E3779B97F4A7C15n);
        let sl=n%11n+1n, sr=63n-n%11n;
        acc=BigInt.asUintN(64,(acc<<sl)|(acc>>sr));
        acc=BigInt.asUintN(64,acc*0xBF58476D1CE4E5B9n);
    }
    let buf=Buffer.alloc(8);
    buf.writeBigUInt64BE(acc,0);
    console.log(buf.toString('base64'));
}
run();

File: bug27_incorrect_formula_used/solved_main.java
import java.nio.ByteBuffer;
import java.util.Base64;

public class solved_main {
    public static void main(String[] args) {
        long acc=0;
        for(long n=1;n<=10;n++){
            long triangle=n*(n+1)/2;   // FIX: was /3; correct triangle formula is n*(n+1)/2
            acc^=triangle*0x9E3779B97F4A7C15L;
            acc=(acc<<(n%11+1))|(acc>>>(63-n%11));
            acc*=0xBF58476D1CE4E5B9L;
        }
        ByteBuffer buf=ByteBuffer.allocate(8);
        buf.putLong(acc);
        System.out.println(Base64.getEncoder().encodeToString(buf.array()));
    }
}
